server/wfprev-api/src/main/java/ca/bc/gov/nrs/wfprev/data/entities/ActivityEntity.java

```diff
@@ -0,0 +1,145 @@
+import org.hibernate.annotations.UuidGenerator;
+import org.springframework.data.annotation.CreatedBy;
+import org.springframework.data.annotation.CreatedDate;
+import org.springframework.data.annotation.LastModifiedBy;
+import org.springframework.data.annotation.LastModifiedDate;
+import org.springframework.data.jpa.domain.support.AuditingEntityListener;
+
+import java.io.Serializable;
+import java.math.BigDecimal;
+import java.util.Date;
+import java.util.UUID;
+
+@Entity
+@EntityListeners(AuditingEntityListener.class)
+@Table(name = "activity")
+@JsonIgnoreProperties(ignoreUnknown = false)
+@Data
+@EqualsAndHashCode(callSuper = false)
+@Builder
+@AllArgsConstructor
+@NoArgsConstructor
+public class ActivityEntity implements Serializable {
+
+    @Id
+    @UuidGenerator
+    @GeneratedValue(strategy = GenerationType.UUID)
+    @Column(name = "activity_guid")
+    private UUID activityGuid;
+
+    @Column(name = "project_plan_fiscal_guid", columnDefinition = "uuid")
+    private UUID projectPlanFiscalGuid;
+
+    @ManyToOne(fetch = FetchType.EAGER, optional = true)
+    @JoinColumn(name = "activity_status_code")
+    private ActivityStatusCodeEntity activityStatusCode;
+
+    @Column(name = "silviculture_base_guid", columnDefinition = "uuid")
+    private UUID silvicultureBaseGuid;
+
+    @Column(name = "silviculture_technique_guid", columnDefinition = "uuid")
+    private UUID silvicultureTechniqueGuid;
+
+    @Column(name = "silviculture_method_guid", columnDefinition = "uuid")
+    private UUID silvicultureMethodGuid;
+
+    @ManyToOne(fetch = FetchType.EAGER, optional = true)
+    @JoinColumn(name = "risk_rating_code")
+    private RiskRatingCodeEntity riskRatingCode;
+
+    @ManyToOne(fetch = FetchType.EAGER, optional = true)
+    @JoinColumn(name = "contract_phase_code")
+    private ContractPhaseCodeEntity contractPhaseCode;
+
+    @Column(name = "activity_funding_source_guid", columnDefinition = "uuid")
+    private UUID activityFundingSourceGuid;
+
+    @NotNull
+    @Column(name = "activity_name", length = 4000)
+    private String activityName;
+
+    @NotNull
+    @Column(name = "activity_description", length = 4000)
+    private String activityDescription;
+
+    @NotNull
+    @Column(name = "activity_start_date")
+    private Date activityStartDate;
+
+    @NotNull
+    @Column(name = "activity_end_date")
+    private Date activityEndDate;
+
+    @Column(name = "planned_spend_amount", precision = 15, scale = 2)
+    private BigDecimal plannedSpendAmount;
+
+    @Column(name = "planned_treatment_area_ha", precision = 15, scale = 4)
+    private BigDecimal plannedTreatmentAreaHa;
+
+    @Column(name = "reported_spend_amount", precision = 15, scale = 2)
+    private BigDecimal reportedSpendAmount;
+
+    @Column(name = "completed_area_ha", precision = 15, scale = 4)
+    private BigDecimal completedAreaHa;
+
+    @NotNull
+    @Column(name = "is_results_reportable_ind", nullable = false)
+    private Boolean isResultsReportableInd;
```

Comment:
I dont find this attribute in the mockup, can you confirm if I are msising this in the figma or it is a value that suppoed to be generate inside API

Reply:
It's in figma 
![image](https://github.com/user-attachments/assets/113d74cd-8092-4a37-af16-59b7e8055af1)